charts/eks-anywhere-packages/templates/registrymirrorsecret.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+data:
```

Comment:
nit, but the ordering of this is wierd.

Usually Data is after kind, and metadata fields when reading k8s files.

Reply:
reordered them. thanks!